test/core/extensions/coreIssuance.spec.ts

```diff
@@ -18,7 +21,15 @@ import {
 import { ether } from '../../../utils/units';
 import { IssuanceComponentDeposited } from '../../../utils/contract_logs/core';
 import { assertTokenBalance, expectRevertError } from '../../../utils/tokenAssertions';
-import { DEFAULT_GAS, DEPLOYED_TOKEN_QUANTITY, ZERO } from '../../../utils/constants';
+import {
+  DEFAULT_GAS,
+  DEPLOYED_TOKEN_QUANTITY,
+  NULL_ADDRESS,
```

Comment:
How is this working? I think you need to rebase, `NULL_ADDRESS` is part of set-protocol-utils now